.github/PULL_REQUEST_TEMPLATE/ontology.md

```diff
@@ -1 +1,8 @@
-## New Ontology Pull Request
\ No newline at end of file
+## New Ontology Pull Request
+
+To propose new Ontology elements, please make sure to **check off each box** below:
+
+- [ ] For new classes and properties an `AnnotationAssertion` `rdfs:comment` and `rdfs:comment` is available.
```

Comment:
I would prefer to phrase this analogously to the other sentences: "I have created..."